From Dean Nelson <dnelson@redhat.com>

Ensure that any errors found by iasl are actually reported by the
ASLTS.  In the past, some were not and therefore just glossed over.

Index: acpica-unix-20140926/tests/aslts/Makefile.def
===================================================================
--- acpica-unix-20140926.orig/tests/aslts/Makefile.def
+++ acpica-unix-20140926/tests/aslts/Makefile.def
@@ -72,7 +72,14 @@ install_all_modes_of_test_case:	${SETOF_
 				"$(ASL)" $$CUR_ASLFLAGS "$(COMMON_ASL_FLAGS)" $(ADD_ASLFLAGS) $$j.asl >> $(COMPILER_LOG) 2>> $(COMPILER_ERROR_LOG); \
 				ret=$$?; \
 				echo "" >> $(COMPILER_LOG); \
-				if [ $$ret != 0 ]; then rval=1; echo "**** Unexpected iASL failure!"; exit 1; fi; \
+                               if [ $$ret != 0 ]; then \
+                                       rval=1; \
+                                       echo "---- Test path: $$dd" >> /dev/stderr; \
+                                       echo "---- Test type: $$CUR_AMLDIR   (Flags $(COMMON_ASL_FLAGS) $$CUR_ASLFLAGS $(ADD_ASLFLAGS))" >> /dev/stderr; \
+                                       tail -2 $(COMPILER_LOG) >> /dev/stderr; \
+                                       echo "**** Unexpected iASL failure!" >> /dev/stderr; \
+                                       exit 1; \
+                               fi; \
 			done; \
 			if [ $$ret != 0 ]; then break; fi; \
 			for j in ${AMLMOD}; do \
